detectron2/export/__init__.py

```diff
@@ -12,4 +12,9 @@
 from .flatten import TracingAdapter
 from .torchscript import scripting_with_instances, dump_torchscript_IR
 
+
+def add_export_config(cfg):
+    return cfg
```

Comment:
The function does nothing and is not used anywhere in detectron2, so there is really no need to call or move it.

It's kept there likely because some internal code calls it - but I'm not sure.

Reply:
I have seen some online examples with this, so probably better to keep it and add a deprecation warning. What do you think?